lib/utils/seqid/src/index.js

```diff
@@ -22,24 +21,49 @@ const pad16 = (t) => {
   return s.slice(s.length - 16) + tt.toString().substr(n.length);
 };
 
-// Return a unique time-based id string
+const validateTimestamp = (timestamp) => {
```

Comment:
This method has strange contract: if it is not valid throws, otherwise just exit. It might be better to return true or false.

Reply:
Before this change `seqid` used to return the next available id, using current time, calculated within. After the change, we allow `seqid` to be used with "outiside" provided sting, which can be anything, by throwing an error here, we are making sure it will not be used in a non-planned or wrong way.
If we make this method return true/false, we would just move the throwing statement.